<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>471</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="nameEdit">
      <property name="placeholderText">
       <string>Имя клиента</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="descriptionEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>200</height>
       </size>
      </property>
      <property name="placeholderText">
       <string>Описание работы</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="phoneEdit">
      <property name="placeholderText">
       <string>Телефон клиента</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="costEdit">
      <property name="placeholderText">
       <string>Стоимость</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="submitButton">
      <property name="text">
       <string>Добавить заказ</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="orderTable">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <column>
       <property name="text">
        <string>Имя</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Описание</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Телефон</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Стоимость</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Статус</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Начало выполнения</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Окончание выполнения</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="searchLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="searchButton">
        <property name="text">
         <string>Поиск</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
